shared_model/interfaces/iroha_internal/query_response_factory.hpp

```diff
@@ -11,7 +11,9 @@
 #include <boost/optional.hpp>
 #include "interfaces/common_objects/account.hpp"
 #include "interfaces/common_objects/asset.hpp"
+#include "interfaces/common_objects/types.hpp"
 #include "interfaces/permissions.hpp"
+#include "interfaces/queries/account_detail_record_id.hpp"
```

Comment:
```suggestion
```

Reply:
https://github.com/hyperledger/iroha/blob/b01d26a1b1e4535dac3d00a2c234e8428756577e/shared_model/interfaces/iroha_internal/query_response_factory.hpp#L69-L74

Reply:
It is included with `types.hpp`, not `account_detail_record_id` interface